packages/discovery-provider/src/api/v1/notifications.py

```diff
@@ -36,11 +36,12 @@
 )
 
 
-@full_ns.route("/<string:user_id>", doc=False)
+@full_ns.route("/<string:user_id>")
 class GetNotifications(Resource):
     @full_ns.doc(
-        id="""Get notifications for user ID""",
+        id="""Get Notifications""",
```

Comment:
I only recently learned that this field drives the name of the endpoint in swagger/generated code and that it has a couple of special behaviors for the transformation.